src/main/scala/org/hammerlab/guacamole/variants/CalledSomaticAllele.scala

```diff
@@ -41,6 +41,9 @@ case class CalledSomaticAllele(sampleName: SampleName,
       .newBuilder
       .setAlleles(seqAsJavaList(Seq(GenotypeAllele.Ref, GenotypeAllele.Alt)))
       .setSampleId(sampleName)
+      .setContigName(contigName)
```

Comment:
i don't know, otomh!

the semantics of the downstream adam and htsjdk models here have changed in this upgrade, so a few things have to specified differently than they were before.

it's also possible that these aren't needed now either; I added them while debugging an issue where variant's INFO fields were coming out blank, which turned out to be due to the `sampleId` not being set consistently throughout.

I can dig in further if you want.